Apps/phgitlab/gitlab.json

```diff
@@ -0,0 +1,450 @@
+                        "123456abcdef"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.id",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "run pipeline",
+            "identifier": "run_pipeline",
+            "description": "Execute a pipeline on a particular branch of a project",
+            "verbose": "Execute a pipeline on a particular branch of a project. GitLab CI/CD must be configured on the project for this action to succeed.",
+            "type": "generic",
+            "read_only": false,
+            "parameters": {
+                "project_id": {
+                    "description": "The ID of the GitLab project",
+                    "data_type": "numeric",
+                    "required": true,
+                    "primary": false,
+                    "contains": [
+                      "gitlab project id"
+                    ],
+                    "order": 0
+                },
+                "trigger_token": {
+                    "description": "The trigger token required for executing a pipeline",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": false,
+                    "contains": [
+                      "gitlab trigger"
+                    ],
+                    "order": 1
+                },
+                "branch": {
+                    "description": "The branch containing the .gitlab-ci.yml file to run. Defaults to 'master'",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": false,
+                    "contains": [
+                      "branch name"
+                    ],
+                    "default": "master",
+                    "order": 2
+                }
+            },
+            "output": [
+                {
+                    "data_path": "action_result.parameter.project_id",
+                    "data_type": "numeric",
+                    "contains": [],
+                    "column_name": "project_id",
+                    "column_order": 0
+                },
+                {
+                    "data_path": "action_result.parameter.trigger_token",
+                    "data_type": "string",
+                    "contains": [],
+                    "column_name": "trigger_token",
+                    "column_order": 1
+                },
+                {
+                    "data_path": "action_result.parameter.branch",
+                    "data_type": "string",
+                    "contains": [],
+                    "column_name": "branch",
+                    "column_order": 3
```

Comment:
There is another value for column_order 3 on line 429

Reply:
I updated the column_order in [this commit](https://github.com/peterbertel/phantom-apps/commit/ca0252ec3d6ed600ef8ebfb3434500fed10ccfa1).